include/odp/arch/arm32-linux/odp/api/abi/cpu.h

```diff
@@ -20,7 +20,11 @@ static inline void odp_cpu_pause(void)
 	 * ISB flushes the pipeline, then restarts. This is guaranteed to
 	 * stall the CPU a number of cycles.
 	 */
-	__asm volatile("isb" ::: "memory");
+#if __ARM_ARCH < 7
+	__asm__ __volatile__("isb" : : : "memory");
+#else
+	__asm__ __volatile__("isb sy" : : : "memory");
+#endif
```

Comment:
I saw error with clang compilation while gcc was fine. I can recheck if this patch now optional or not.

Reply:
@muvarov Well, I'm just not sure how does changing M4 macros affect compilation. Could you please recheck? Because if you saw this error (btw, could you please include it into commit message) after changing macros, it can be a sign of some hidden behavioural change.